keentools_facebuilder/interface/panels.py

```diff
@@ -535,6 +535,8 @@ def draw(self, context):
         row = box.row()
         row.prop(head, 'model_scale')
 
+        box.prop(head, 'model_level')
```

Comment:
Not sure about the name. It's not level only, maybe there will be other models (e.g. DAZ model)